redfish-client/redfish-client

```diff
@@ -24,7 +24,7 @@ redfish-client ::
    --insecure           Ignore SSL certificates
    --debug LEVEL        Run in debug mode, LEVEL from 1 to 3 increase verbosity
                         Security warning LEVEL > 1 could reveal password into the logs
-   --debugfile FILE     Specify the client debugfile [default: redfish-client.log]
+   --debugfile FILE     Specify the client debugfile [default: /var/log/python-redfish/redfish-client.log]
```

Comment:
It would be good in the future to also add an entry in the config file and use the value from it by default, the option on the CLI allowing to overwrite that.